handlers/About/components/Leader/index.js

```diff
@@ -0,0 +1,38 @@
+/** @flow */
+
+require('./styles.css');
+
+import React from 'react';
+import lookup from 'lookup';
+import {Link} from 'react-router';
+
+var {PropTypes} = React;
+
+class Leader extends React.Component {
+
+  render(): ?ReactElement {
+    var leader = this.props.leader;
+    var index = this.props.index;
+    var floatClass = index % 2 ? ' is-odd' : '';
+    return (
+      <li className="Leader">
+       <div className={`Leader-image-container${floatClass}`}>
+        <img className="Leader-image" src={lookup(leader, 'headShot.fields.file.url')} />
+      </div>
+      <div className={`Leader-meta ${floatClass}`}>
+      <h3 className="Leader-name">{leader.firstName} {leader.lastName}</h3>
```

Comment:
Is there a reason to use `<h3>` for this rather than a `<span>` with a class? It doesn't appear to be a site hierarchy use.